crates/sui-framework/tests/math_tests.move

```diff
@@ -25,6 +25,11 @@ module sui::math_tests {
         while (i < 1000) {
             assert!(math::sqrt(i * i) == i, 1);
             i = i + 1;
+        };
+        let i = 0xFFFFFFFFFu128;
```

Comment:
Should have a test that makes sure this works for values that fit in u128 but not u64.

Reply:
Right. Didn't want to mix too many unrelated things in this PR. Fleshing out `math.move` should happen separately.